packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -280,19 +280,19 @@ class HorizontalOverflow extends Foundation<
      */
     private getPositionData = (): PositionChange => {
         const scrollPosition: number = this.getScrollPosition();
-        const distanceRemaining: number =
-            this.horizontalOverflowItemsRef.current.scrollWidth - scrollPosition;
 
-        const scrollPositionIsAtBeginning: boolean = scrollPosition === 0;
-        const scrollPositionIsAtEnd: boolean =
-            distanceRemaining === this.horizontalOverflowItemsRef.current.clientWidth;
-        const scrollPositionIsInMiddle: boolean =
-            !scrollPositionIsAtEnd && !scrollPositionIsAtBeginning;
+        const isAtBeginning: boolean = scrollPosition === 0;
+        const isAtEnd: boolean =
+            this.horizontalOverflowItemsRef.current.scrollWidth - scrollPosition ===
+            this.horizontalOverflowItemsRef.current.clientWidth;
 
-        return {
-            start: scrollPositionIsInMiddle ? false : scrollPositionIsAtBeginning,
-            end: scrollPositionIsInMiddle ? false : scrollPositionIsAtEnd,
-        };
+        if (isAtBeginning) {
+            return { start: false, end: true };
+        } else if (isAtEnd) {
+            return { start: true, end: false };
```

Comment:
seems backwards, shouldnt start be `false` and end be `true`? and vice versa above?

Reply:
If you are ate the beginning there is nothing to scroll to going left, so it returns false, and there is content to scroll to on the end side, so true.  I agree this terminology is confusing.

Reply:
it seems to me all of the expected values are flipped, see my previous comment on the documentation that dictates what start and end represent.